ci/ray_ci/test_windows_container.py

```diff
@@ -36,23 +37,16 @@ def _mock_subprocess(inputs: List[str], stdout, stderr) -> None:
 
 def test_get_run_command() -> None:
     container = WindowsContainer("test")
+    envs = []
+    for env in _DOCKER_ENV:
```

Comment:
does linux have the same thing? why does the linux one do not have this issue?

Reply:
linux has BUILDKITE=true set inside the base docker image